Get-help.md

```diff
@@ -0,0 +1,141 @@
+
+### How to ask a general question
+
+Follow the template below for asking questions (fill in the values inside {{}} brackets and mark x inside checkboxes to tick them) to leave a comment on a pull request. Adapt the template as needed if you are using another channel.
+
+```md
+@{{PR reviewer or Mentor username}} PTAL!
+
+**Checklist**
+- [ ] I have filled the [CLA](https://goo.gl/forms/AttNH80OV0) and the
+  [Oppia Contributor Survey](https://goo.gl/forms/otv30JV3Ihv0dT3C3)
+- [ ] I have setup Oppia locally (verified by running `python -m
+  scripts.start`)
+- [ ] I have worked through the "Before you ask a question" section
+
+**System Information**
+- OS: (Be specific Ex: Ubuntu 20.04)
+- [ ] Virtual machine
+
+**About the issue**
+- #{{Issue_No}}
+- #{{PR_No}}
+- {{Describe the issue}} (eg. Failing e2e tests: "The test has been
+  failing repeatedly since {{x}} previous runs, and I haven’t done any
+  changes related to it. Requesting a re-run.")
+
+**Approaches already used to resolve the issue**
+
+(eg: Link to a Stack Overflow answer or any solution that you have tried)
+- #{{Link to the debugging doc}}
+- enter any additional description
+```
+
+### Important points
+
+* If you are unable to push changes due to some reason, you can create a [patch file](https://docs.gitlab.com/omnibus/development/creating-patches.html) and share it with the person you're asking for help.
+
+* If you are facing issues in completing the assigned task, you can create a PR on your fork of the Oppia repository, troubleshoot your problem on that pull request with help from your mentor, and then create a new PR on the original Oppia repository.
+
+* If you have not made a PR yet, because you are not sure:
+
+  * what the issue is about, or
+  * which files have to be modified, or
+  * if your approach towards the solution is correct
+
+  Then ask for help by commenting with your doubt/suggested approach on the issue page itself.  If you don’t get any response within **24 hours**, you can drop a message on the [gitter](https://gitter.im/oppia/oppia-chat#) chat too.
+
+* If you want to have a discussion on your approach, but aren’t ready to make a PR yet, you can create a [public gist](https://gist.github.com/) and include the link to it in your question. It’s always better to see the code you are talking about!
+
+* **Avoid asking for help from people via emails or direct messaging.** We encourage everyone to ask for help on a common channel so that whoever sees your query first can help you or guide you how to take your query forward. Note that directing questions to your mentor is fine.
+
+  * Comment on the issue page or the PR if it’s very specific,
```

Comment:
Not clear what "it" refers to here (also not sure what a PR being very specific means).

Reply:
done